code-reviews/vowel-counter/vowel-counter.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+def cnt_vowels(x):
+	y = "aeiou"
+	n = 0
+	for v in y:
+		n += x.count(v)
+	return n
+def cnt_consonants(m):
+	x = "aeiou"
+	for l in m:
+		if l not in x:
+			print(f"consonant={l}")
```

Comment:
You should collect the consonant count and return that instead of printing it out here.